Dockerfile

```diff
@@ -13,6 +13,10 @@ RUN curl -fsSL https://github.com/krallin/tini/releases/download/v${TINI_VERSION
         --output /tini \
     && chmod +x /tini
 
+RUN curl -fsSL https://github.com/dmikusa/tiny-health-checker/releases/download/v0.33.0/thc-x86_64-unknown-linux-musl \
```

Comment:
We support ARM too.

Reply:
>We support ARM too. And why not using a common tool like wget or curl?

I wanted to use something simple with a small attack surface, curl works but seems to be overkill IMO.

Reply:
I have a better idea. Have the "healthcheck" command builtin into the invidious companion binary.

Add a new helper in the helper folder and if you run "./invidious companion healthcheck" then this will ping /health and confirm if it works or not.

This way, no additional binary!

Reply:
>I have a better idea. Have the "healthcheck" command builtin into the invidious companion binary.

That is what I wanted to do in the first place! But as I said in https://github.com/iv-org/invidious-companion/pull/27#issuecomment-2552362347, I couldn't find a way to make it run synchronously without executing none of the code that is bellow. It still works tho so I will add what I did in https://github.com/Fijxu/invidious-companion/commit/6c40b8a9354e207b375be2bb5b8ce0cc15a96744

Reply:
Ha yes indeed right! I think it's time I introduce some kind of args system. Like having `--version`, `--run` and so on. I have created a github issue: https://github.com/iv-org/invidious-companion/issues/33

This way we can add the arg `healthcheck` which do not execute the main code but instead just check if a local install is running.

Could you please remove the modifications about the Dockerfile? We will see about these modifications in another PR which will add the args system.

Thanks.

Reply:
Done!